lightfm/evaluation.py

```diff
@@ -135,8 +138,9 @@ def auc_score(model, test_interactions, train_interactions=None,
               user_features=None, item_features=None,
               preserve_rows=False, num_threads=1):
     """
-    Measure the ROC AUC metric for a model: the probability that a randomly chosen positive
-    example has a higher score than a randomly chosen negative example.
+    Measure the ROC AUC metric for a model: the probability that a randomly 
```

Comment:
Can you have a look at the CircleCI build results and address the lint violations?